File: conexion_db_go/conexion_postgres.go
package main

import (
	"database/sql"
	"fmt"
	"log"

	_ "github.com/lib/pq"
)

func conectar_postgres() {
	connStr := "user=tu_usuario password=tu_contraseña dbname=tu_bd host=localhost port=5432 sslmode=disable"
	db, err := sql.Open("postgres", connStr)
	if err != nil {
		log.Fatal(err)
	}
	defer db.Close()

	err = db.Ping()
	if err != nil {
		log.Fatal("No se pudo conectar a PostgreSQL:", err)
	}

	fmt.Println("Conexión exitosa a PostgreSQL")
}

File: conexion_db_javascript/conexion_postgres.js
const { Client } = require("pg");

const client = new Client({
  user: "tu_usuario",
  host: "localhost",
  database: "tu_base_de_datos",
  password: "tu_contraseña",
  port: 5432,
});

async function conectarPostgres() {
  try {
    await client.connect();
    console.log("Conexión exitosa a PostgreSQL");
  } catch (error) {
    console.error("Error de conexión:", error);
  } finally {
    await client.end();
  }
}

conectarPostgres();
